tests/RobotFramework/tests/MQTT_health_check/health_tedge-mapper-az.robot

```diff
@@ -46,3 +46,21 @@ Stop watchdog service
 
 Remove entry from service file
     Execute Command    sudo sed -i '10d' /lib/systemd/system/tedge-mapper-az.service
+
+Watchdog does not kill mapper if it responds
+    # Set the watchdog interval low so we don't have to wait long
+    Execute Command    sudo systemctl stop tedge-mapper-az.service
+    Execute Command    sudo systemctl stop tedge-watchdog.service
+    Execute Command    cmd=sudo sed -i '10iWatchdogSec=5' /lib/systemd/system/tedge-mapper-az.service
+    Execute Command    sudo systemctl daemon-reload
+    Execute Command    sudo systemctl start tedge-mapper-az.service
+    Execute Command    sudo systemctl start tedge-watchdog.service
+
+    ${pid_before_healthcheck}=    Execute Command    pgrep -f '^/usr/bin/tedge-mapper az'    strip=${True}
+    # The watchdog should send a health check command while we wait
+    Sleep    10s
+    ${pid_after_healthcheck}=     Execute Command    pgrep -f '^/usr/bin/tedge-mapper az'    strip=${True}
+
+    Should Have MQTT Messages     topic=te/device/main/service/tedge-mapper-az/cmd/health/check    minimum=1
+    Should Be Equal               ${pid_before_healthcheck}    ${pid_after_healthcheck}
+
```

Comment:
Don't we need such tests for c8y mapper?

Reply:
Added an analogous test for c8y mapper in next PR: 8f2727d0892daa3d8cc0df07b2e4aa7094cb14e5